napari_tiff/napari_tiff_metadata.py

```diff
@@ -2,14 +2,9 @@
 
 import numpy
 from tifffile import PHOTOMETRIC, TiffFile, xml2dict
-from vispy.color import Colormap
+from napari.utils.colormaps import ALL_COLORMAPS, Colormap
```

Comment:
napari allows passing colormap as name, or array of colors. So this import is obsolete 

All are valid: 
```python
viewer.layers[0].colormap= "magenta"

viewer.layers[0].colormap = [(0,0,0,0), (1, 0, 0, 1)]

viewer.layers[0].colormap = np.array([(0,0,0,0), (0, 1, 0, 1)])
```